daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -155,17 +155,23 @@ Learn more in the [declarative and programmatic subscriptions doc]({{< ref subsc
 Create a file named `subscription.yaml` and paste the following:
 
 ```yaml
-apiVersion: dapr.io/v1alpha1
+apiVersion: dapr.io/v2alpha1
```

Comment:
I would not just blanket replace this. See the example image that I attached to the issue and modify the existing example to match this. The only real change is the apiVersion and the route information. I would remove all the rules, since these are confusing.